### PR TITLE
Flutter platform spec usage

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

**Comment by sigurdm** [comment]
cc @mit-mit

File: lib/src/pubspec.dart [hunk 1]
```diff
@@ -112,6 +112,15 @@ class Pubspec {
 
   SdkConstraintStatus get sdkConstraintStatus =>
       SdkConstraintStatus.fromSdkVersion(_inner.environment['sdk']);
+
+  bool get usesOldFlutterPluginFormat =>
+      usesFlutter &&
+      _inner.flutter['plugin'] != null &&
+      const {
```
Comment (isoos) [inline lib/src/pubspec.dart]: `_inner.flutter['plugin'] is Map &&`

Comment (sigurdm) [reply lib/src/pubspec.dart]: Good catch.
Done

**Comment by sigurdm** [comment]
We should only land this after support for the new format is in flutter/stable

**Comment by sigurdm** [comment]
@jonasfj we are ready to land this now, right?

**Pull request merged**